src/anm/animation/scene.js

```diff
@@ -101,6 +105,20 @@ Scene.prototype.reverseEachVisible = function(visitor, data) {
     }
 };
 
+Scene.prototype.findDeepestChildAt = function(local_pt) {
```

Comment:
Scene should be made element (at some point, not now) at least to avoid this code dups